src/data/script.ts

```diff
@@ -436,6 +436,10 @@ export const hasScriptFields = (
 export const migrateAutomationAction = (
   action: Action | Action[]
 ): Action | Action[] => {
+  if (!action) {
+    return [];
```

Comment:
```suggestion
    return action;
```

Reply:
It should be alright to return an empty array, since we also do it on a different place: 

https://github.com/home-assistant/frontend/blob/6a337cc486c637ddff55f746ac3769819bab2946/src/components/ha-selector/ha-selector-trigger.ts#L24-L25

Reply:
That is a different situation I would say, as there an array is needed for the element to work.

In the convert script we don't want to change any config except for what we explicitly have defined.

Reply:
Alright, I understand. It's changed.